corehq/apps/users/forms.py

```diff
@@ -1214,12 +1214,10 @@ def __init__(self, *args, **kwargs):
             (role._id, role.name or _('(No Name)')) for role in roles
         ]
 
-        self.fields['domains'].choices = [(self.domain, self.domain)]
-        if len(DomainPermissionsMirror.mirror_domains(self.domain)) > 0:
-            self.fields['domains'].choices = [('all_project_spaces', _('All Project Spaces'))] + \
-                                             [(self.domain, self.domain)] + \
-                                             [(domain, domain) for domain in
-                                              DomainPermissionsMirror.mirror_domains(self.domain)]
+        self.fields['domains'].choices = [('all_project_spaces', _('All Project Spaces'))] + \
+                                         [(self.domain, self.domain)] + \
+                                         [(domain, domain) for domain in
+                                          DomainPermissionsMirror.mirror_domains(self.domain)]
```

Comment:
just to understand this better,

1. was `[(self.domain, self.domain)]` twice earlier in case there were mirror domains?
2. how does this impact the UI? it made sense earlier when we were only show options here if needed. Now they would see "All Project Spaces" and the domain each time.

Reply:
This does change the UI. Previously, the "Project Spaces" filter was always displayed, but if there weren't mirror domains, it would only ever contain the current domain. Now the filter is only displayed if there are mirror domains.

Reply:
So we are populating choices always, but just showing it only when needed. That sounds okay.